app/controllers/bars_controller.rb

```diff
@@ -1,3 +1,6 @@
+require "mailgun"
```

Comment:
Rails actually loads all gems inside of itself. You don't need to `require` anything at all!